app/services/events/stores/aggregated_clickhouse_store.rb

```diff
@@ -205,6 +186,33 @@ def to_decimal128(value)
       def cast_to_json(attribute)
         Arel::Nodes::SqlLiteral.new("#{attribute.relation.name}.#{attribute.name}::JSON")
       end
+
+      def merge_aggregation(aggregation_type, column, as)
+        connection_with_retry do |connection|
+          sql = aggregated_events_sql(select: [
+            to_decimal128(Arel::Nodes::NamedFunction.new(
+              aggregation_type,
+              [aggregated_arel_table[column]]
+            )).as(as)
```

Comment:
Also just realized we might not even need to specify `.as` since we retrive by index and not by column name/alias.